azure_img_utils/cloud_partner.py

```diff
@@ -27,6 +27,8 @@
 from azure_img_utils.exceptions import AzureCloudPartnerException
 from requests.exceptions import HTTPError
 
+ingestion_api = 'https://graph.microsoft.com/rp/product-ingestion/'
```

Comment:
The scope of this variable might be clearer in CAPS?

Reply:
Done